Shared/Realm.Sync.Shared/ProgressNotifications/ProgressDirection.cs

```diff
@@ -0,0 +1,36 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2017 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+namespace Realms.Sync
+{
+    /// <summary>
+    /// The transfer direction (upload or download) tracked by a given progress notification subscription.
+    /// </summary>
+    public enum ProgressDirection : byte
+    {
+        /// <summary>
+        /// Monitors upload progress.
+        /// </summary>
+        Upload,
```

Comment:
This maps directly to the C++ enum, right? I realize that we are not going to neither change the order of these, nor add a third direction but I would still be more comfortable specifying the value explicitly.